src/NuGet.Core/NuGet.ProjectModel/LockFile/LockFileFormat.cs

```diff
@@ -28,7 +28,7 @@ public class LockFileFormat
         private const string LibrariesProperty = "libraries";
         private const string TargetsProperty = "targets";
         private const string ProjectFileDependencyGroupsProperty = "projectFileDependencyGroups";
-        private const string ServicableProperty = "servicable";
+        private const string ServiceableProperty = "serviceable";
```

Comment:
This might cause issue. @nkolev92 Is it safe to change this property name?

Reply:
Hey @sify21.

Thank you so much for the issue and the code contribution.
It is unfortunate that we have this typo in our formatter code.
Unfortunately we cannot take this change because the assets file is used for some performance optimizations, and this would be a breaking change.